src/config/ch.sbb.netzkarte/index.js

```diff
@@ -305,6 +307,30 @@ export const gemeindegrenzen = new MapboxStyleLayer({
   },
 });
 
+export const geschosseLayer = new Layer({
+  name: "ch.sbb.geschosse",
+  visible: true,
+  properties: {
+    hideInLayerTree: true,
+  },
+});
+
+geschosseLayer.children = FLOOR_LEVELS.map((level) => {
+  return new LevelLayer({
+    name: `ch.sbb.geschosse${level}`,
+    visible: level === "2D",
+    mapboxLayer: dataLayer,
+    styleLayersFilter: ({ metadata }) => metadata && metadata["geops.filter"],
+    level,
+    group: "ch.sbb.geschosse-layer",
+    properties: {
+      parent: geschosseLayer,
+      hideInLayerTree: true,
+      baselayers: [netzkarteLayer, netzkarteNight],
```

Comment:
This property baselayers is hardly understandable .  Can we use a property `hasLevels`  on the base layers instead ?  and in the floorSwitcher you just test this property to display the switcher.

Reply:
Yeah makes sense